<launch>
  <node
    pkg="skyrats_cbr_2021"
    type="display_recog.py"
    name="display_recognition"
    output="screen"
  /> 
  <node
    pkg="skyrats_cbr_2021"
    type="MRS_MAV.py"
    name="MRS_MAV"
    output="screen"
  />
</launch> 